include/seastar/core/internal/stall_detector.hh

```diff
@@ -59,6 +60,7 @@ protected:
     std::atomic<uint64_t> _stall_detector_missed_ticks = { 0 };
     unsigned _reported = 0;
     unsigned _total_reported = 0;
+    metrics::time_estimated_histogram _stalls_histogram;
```

Comment:
You're not using approx_blah_historgram here.

Reply:
@avikivity right, but time_estimated_histogram uses it.

I've took it as-is from scyllaDB repository, in the future we can decide to keep only this copy